core/accounts/templates/password_reset_done.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block title %}Email Sent{% endblock %}
+
+{% block body %}
+<div>
+  <h1>Check your inbox.</h1>
+  <p>We've emailed you instructions for setting your password. You should receive the email shortly!</p>
```

Comment:
Should be: “An email has been sent to the address provided. Please follow the instructions within to reset your UWA Case password.”